app/scripts/controllers/transactions/index.js

```diff
@@ -1502,6 +1510,21 @@ export default class TransactionController extends EventEmitter {
       gasParams.gas_price = gasPrice;
     }
 
+    if (defaultGasEstimates) {
+      const estimateType = defaultGasEstimates.defaultEstimate;
+      const { gasFeeEstimates } = await this._getEIP1559GasFeeEstimates();
+      gasParams.default_gas_estimates = this._getGasValuesInGWEI({
```

Comment:
[REQ] --- this can't be an object, has to be nested properties. gasParams will be passed to _getGasValuesInGWEI on line 1536. Just add the values directly to gasParams here.
gasParams.default_gas_price, gasParams.default_max_priority_fee_per_gas, etc.

Reply:
I see just fixed